docs/modules/sql/pages/mapping-to-jdbc.adoc

```diff
@@ -19,6 +19,8 @@ The JDBC connector supports the following database management systems:
 
 NOTE: You can use other database management systems with a JDBC interface. However, Hazelcast offers no guarantees or support for these systems.
 
+NOTE: Oracle converts table and column names to upper case if they are not put inside double quotes, otherwise the exact case is kept. Either put the table and column names in double quotes or refer to them in upper case later (e.g. when used in Hazelcast mapping).
```

Comment:
```suggestion
NOTE: If your table and column names include lower case characters, you must enclose the names in double quotes to prevent Oracle converting the names to upper case. If you do not enclose the table and column names in double quotes, use upper case when referring to them, for example, in Hazelcast mapping.
```